pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/options/impl/DisableRestartFromStage.java

```diff
@@ -0,0 +1,28 @@
+package org.jenkinsci.plugins.pipeline.modeldefinition.options.impl;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import org.jenkinsci.Symbol;
+import org.jenkinsci.plugins.pipeline.modeldefinition.options.DeclarativeOption;
+import org.jenkinsci.plugins.pipeline.modeldefinition.options.DeclarativeOptionDescriptor;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+public class DisableRestartFromStage extends DeclarativeOption {
+
+    @DataBoundConstructor
+    public DisableRestartFromStage() {}
```

Comment:
Did you check whether this works with the directive generator (open a Pipeline job in Jenkins, then click "Pipeline Syntax" in the sidebar and then "Declarative Directive Generator")? I cannot remember if Jelly files need to be added for options like this or not. Since this option does not have any configuration, it might be ok as-is.

Reply:
No i haven't. Let me do that and i'll get back to you

Reply:
Hmmmm. I can't add an option at all. Not even when i checkout master branch, rebuild and rerun Jenkins... Not sure why at this point. Build was done both with java 8 and then java 11

Reply:
Looks like i will have to add Jelly files. Still figuring out